src/main/java/org/symphonyoss/integration/webhook/github/parser/v1/BaseDeploymentGithubParser.java

```diff
@@ -69,11 +69,14 @@
   private static final String PRESENTATIONML_MESSAGE_DESCRIPTION = "<br/>Description: %s";
 
   @Override
-  public String parse(Map<String, String> parameters, JsonNode node) throws GithubParserException {
+  protected String getMessage(Map<String, String> parameters, JsonNode node)
+      throws GithubParserException {
     try {
-      return buildEntityML(node);
-    } catch (EntityXMLGeneratorException | URISyntaxException e) {
-      throw new GithubParserException("Something went wrong while building the message for Github", e);
+      String entityML = buildEntityML(node);
+      return entityML;
```

Comment:
The variable entityML isn't required, you might return directly.

Like this: return buildEntityML(node);